score-api/src/main/java/io/cloudslang/score/lang/ExecutionRuntimeServices.java

```diff
@@ -83,6 +83,14 @@
 
     private static final String WORKER_GROUP_NAME = "WORKER_GROUP_NAME";
 
+    private static final String CONSUMER_WORKER_ID = "CONSUMER_WORKER_ID";
+
+    private static final String PRODUCER_WORKER_ID = "PRODUCER_WORKER_ID";
+
+    private static final String ROBOT_ID = "ROBOT_ID";
+
+    private static final String ACTIVITY_GROUP = "ACTIVITY_GROUP";
```

Comment:
Think about renaming to ROBOT_GROUP_NAME to keep consistency with WORKER_GROUP_NAME